source/introduction.rst

```diff
@@ -22,30 +22,26 @@ easy to remember namespace for their projects.
 Who is part of the PyCQA?
 =========================
 
-The projects housed
-under the PyCQA are all presently maintained by the following people:
-
-- `Astroid <https://github.com/pycqa/astroid>`_
-
-  * `Ashley Whetter <https://github.com/awhetter>`_
-
-  * `ceridwen <https://github.com/ceridwen>`_
-
-  * `Claudiu Popa <https://github.com/PCManticore>`_
-
-  * `Dmitry Pribysh <https://github.com/dmand>`_
-
-  * `Florian Bruhin <https://github.com/The-Compiler>`_
-
-  * `Guillaume Peillex <https://github.com/hippo91>`_
-
-  * `Łukasz Rogalski <https://github.com/rogalski>`_
-
-  * `jcristau <https://github.com/jcristau>`_
-
-  * `Roy Williams <https://github.com/rowillia>`_
-
-  * `Sylvain Thénault <https://github.com/sthenault>`_
+The projects housed under the PyCQA are all presently maintained by the
+following people:
+
+- `Astroid <https://github.com/pycqa/astroid>`_ and `Pylint <https://github.com/pycqa/pylint>`_
+    - Maintainers:
+          * `Ashley Whetter <https://github.com/awhetter>`_
+          * `Bryce Guinta <https://github.com/brycepg>`_
+          * `Guillaume Peillex <https://github.com/hippo91>`_
+          * `Pierre Sassoulas <https://github.com/Pierre-Sassoulas>`_
+          * `Claudiu Popa <https://github.com/PCManticore>`_
+    - Developers:
+          * `Marc Mueller <https://github.com/cdce8p>`_
+          * `ceridwen <https://github.com/ceridwen>`_
+          * `Dmitry Pribysh <https://github.com/dmand>`_
+          * `jcristau <https://github.com/jcristau>`_
+          * `Sushobhit <https://github.com/sushobhit27>`_
+          * `Florian Bruhin <https://github.com/The-Compiler>`_
+          * `Łukasz Rogalski <https://github.com/rogalski>`_
+          * `Roy Williams <https://github.com/rowillia>`_
+          * `Sylvain Thénault <https://github.com/sthenault>`_
```

Comment:
Please follow the style of the rest of the file. (new lines between list elements)

Reply:
Fixed